docs/zdgbook/ObjectPublishing.rst

```diff
@@ -826,103 +826,101 @@ Method Arguments
 ----------------
 
 Sometimes you may wish to control which object is published based on
-form data. For example, you might want to have a form with a select
```

Comment:
the old line was correct

Reply:
Sorry - pulled the review button to early - this change was done in the other pull request - and fixed there already.